ibi/coreos-installer/manifest.template

```diff
@@ -0,0 +1,2 @@
+{"version":"4.14.5","domain":"REPLACE_DOMAIN","cluster_name":"REPLACE_CLUSTER_NAME","cluster_id":"REPLACE_CLUSTER_ID","master_ip":"REPLACE_MASTER_IP","release_registry":"quay.io","hostname":"REPLACE_HOSTNAME"}
```

Comment:
If we want to use envsubst, we should change `REPLACE_WHATEVER` with `${WHATEVER}` or something like that